docs/updating.rst

```diff
@@ -2,35 +2,55 @@
 How to update the Atomic Developer Bundle to the latest version
 ===============================================================
 
-* If you are using the image from atlas.hashicorp.com
+Depending on your implementation of the ADB, you might be able to directly
+update your Vagrant box. In some cases, you will need to perform a clean
+installation or follow additional steps to complete the update.
```

Comment:
We need a prerequisite statement that says that you must not only `vagrant halt` your existing ADB, but you must actually `vagrant destroy` it.

This will lose any data or configuration in your ADB that is not saved on your host.

@budhrg please confirm.